docs/eog/case-studies-cw20-agenda.rst

```diff
@@ -0,0 +1,76 @@
+.. _CW20-Agenda: 
+
+CW20 Agenda
+============
+
+For in-person CWs, the days are long to maximise the opportunities for networking, collaboration, discussions, and solving problems across the research software landscape. 
+The agenda that we had spent months organising was packed full of talks, group sessions, social activities and mini-workshops - we originally had 18 mini-workshops scheduled to take place across five sessions.
+
+If we had organised CW20 as an online event from the beginning, there are many things we would have done differently - but since we found ourselves in the position of having to transition from an in-person event to an online one, we still wanted to incorporate as much of the original programme as possible. 
+We contacted speakers and workshop facilitators to ask if they were still able to deliver in the online setting. 
+In the end, two of the workshops dropped out and we were therefore able to shift the others around so that we had 16 workshops across four sessions (with four workshops in parallel for each session). 
+
+Because we were now unable to carry out the planned social programme (activities such visiting the Ulster Museum, a walk around the Belfast Botanic Gardens and of course the conference dinner), we were able to shorten the days and add in further breaks and changeover/buffer time. 
```

Comment:
```suggestion
Because we were now unable to carry out the planned social programme (activities such visiting the Ulster Museum, a walk around the Belfast Botanic gardens and of course the conference dinner), we were able to shorten the days and add in further breaks and changeover/buffer time. 
```